apps/features/migrations/0006_suite_features_naming.py

```
@@ -0,0 +1,21 @@
+# Generated by Django 5.2.10 on 2026-02-06 22:02
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("features", "0005_update_standard_charge_point"),
+    ]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)